docs/tutorials/python-api.md

```diff
@@ -98,11 +97,8 @@ from ragna.assistants import RagnaDemoAssistant
 ```
 
 !!! note
-
-    The RagnaDemoAssistant is not an assistant(LLM), instead it replies with the
-    your prompt and a static message. It is only to understand the Ragna API. You need
-    to get API keys and set relevant environment variables to use the supported
-    assistants.
+    You need to [get API keys](../references/faq.md#where-do-i-get-api-keys-for-the-builtin-assistants)
```

Comment:
We need to keep the newline here

```suggestion

    You need to [get API keys](../references/faq.md#where-do-i-get-api-keys-for-the-builtin-assistants)
```

This seems to render correctly, but will no longer after the auto-formatter is done with the file: https://github.com/Quansight/ragna/actions/runs/6835971543/job/18591266659?pr=196#step:6:65

Reply:
[added back](https://github.com/Quansight/ragna/pull/196/commits/576d96eb59c66245ac8611177e9d74a842b317c9) the new line